src/query/service/src/sql/planner/binder/copy.rs

```diff
@@ -562,7 +562,10 @@ pub async fn parse_stage_location_v2(
     name: &str,
     path: &str,
 ) -> Result<(UserStageInfo, String)> {
-    debug_assert!(path.starts_with('/'), "path should starts with '/'");
+    if !path.starts_with('/') {
+        let e = ErrorCode::SyntaxException("path should starts with '/'");
```

Comment:
Don't expose internal logic to users. I use `debug_assert` here because I expect input path must be `starts_with('/')`. If not, something must be wrong.

Please check the caller of `parse_stage_location_v2` and add unit tests for them to make it sure.